app/src/main/java/com/example/parkingsystemjava/mvp/view/ParkingView.java

```diff
@@ -24,13 +26,23 @@ public ParkingView(Activity activity, ActivityMainBinding binding) {
     public void showParkingsAvailable(int parkingsAvailable) {
         FragmentManager fragmentManager = getFragmentManager();
         DialogFragmentParkingLots editNameDialogFragment = new DialogFragmentParkingLots();
-        editNameDialogFragment.show(fragmentManager, DIALOG_FRAGMENT_TAG);
+        editNameDialogFragment.show(fragmentManager, ConstantUtils.DIALOG_FRAGMENT_TAG);
     }
 
+    @Override
     public void showMessageOfParkingLots(int parkingsAvailable) {
         Context context = getContext();
         if (context != null) {
             Toast.makeText(context, context.getString(R.string.toast_main_parking_size, parkingsAvailable), Toast.LENGTH_SHORT).show();
         }
     }
+
+    public void showReservationScreen() {
+        Context context = getContext();
+        Activity activity = getActivity();
+        if (context != null && activity != null) {
+            Intent intent = ReservationActivity.newInstance(context);
+            activity.startActivity(intent);
```

Comment:
here too, you can do this 
`activity.startActivity(ReservationActivity.newInstance(context););`

Reply:
Done